lib/modules/hideSidebar.js

```diff
@@ -0,0 +1,116 @@
+/* @flow */
+import { $ } from '../vendor';
+import { Module } from '../core/module';
+import { i18n } from '../environment';
+import * as Menu from './menu';
+import {
+	CreateElement
+} from '../utils';
+
+export const module: Module<*> = new Module('hideSidebar');
+
+module.moduleName = 'hideSidebarName';
+module.category = 'productivityCategory';
+module.description = 'hideSidebarTabDesc';
+
+let hideSidebar;
+let propToChange;
+let valueToChangeTo;
+let valueToChangeFrom;
+let $side;
+let $content;
+let contentOffset;
+let headerHeight;
+let documentHalf;
+let sideContent;
+let sideWidth;
+let rulesCheck = ['reddiquette', 'rules'];
+
+const toggleSidebar = () => {
+
+	if (!$content.data('sidebar-hidden')) {
```

Comment:
Any particular reason to store this in `$.data` instead of a variable here in the module?

Reply:
Done.